source/clear-linux/tutorials/gce/gce.rst

```diff
@@ -0,0 +1,122 @@
+   a single global namespace.
+
+   .. figure:: figures/01-cloud-storage.png
+      :scale: 50 %
+      :alt: Browse Google Cloud Storage
+
+   .. figure:: figures/02-storage-bucket.png
+      :scale: 50 %
+      :alt: Create a bucket in the Cloud Storage
+
+   .. figure:: figures/03-create-bucket.png
+      :scale: 50 %
+      :alt: Set an unique bucket name
+
+#. Once the bucket is created, press the ``Upload files`` button
+   on the Bucket details page to upload the |CL| GCE image archive
+   to the named bucket:
+
+   .. figure:: figures/04-bucket-created.png
+      :scale: 50 %
+      :alt: Cloud Storage bucket is available for storing objects
+
+   .. figure:: figures/10-image-upload.png
+      :scale: 50 %
+      :alt: Uploading the image source archive file
+
+   .. figure:: figures/11-bucket-uploaded.png
+      :scale: 50 %
+      :alt: Image archive imported complete
+
+#. Click the *Navigation menu* icon on the upper left screen menu,
+   mouse hovers over the *Compute Engine* menu group and
+   select the *Images* item.
+   On the Compute Engine Images page, click the ``[+] CREATE IMAGE`` menu
+   item, change the image source type to *Cloud Storage file*,
+   and pick up the image archive object in the named bucket by clicking
+   the ``Browse`` button of the *Cloud Storage file* input box.
+   Accept the rest of default options, and click the ``Create`` button
+   at the bottom to add a Clear Linux GCE image to the image list.
+
+   .. figure:: figures/20-gce-image.png
+      :scale: 50 %
+      :alt: Create a Google Compute Engine image
+
+   .. figure:: figures/21-create-image.png
+      :scale: 50 %
+      :alt: Create the image using the imported image archive object
+
+   .. figure:: figures/22-image-list.png
+      :scale: 50 %
+      :alt: Clear Linux Compute Engine image is created
+
+#. After the |CL| image is imported, you can launch a VM instance running
+   |CL| by following the instructions documented in the
+   `Quick Start <https://cloud.google.com/compute/docs/quickstart-linux>`_
+   guide, except the step of selecting *Boot disk* --
+   Instead of using the stock OS images, click the *Custom images* tab
+   to select the imported Clear Linux OS GCE image.
+
+   .. figure:: figures/30-create-vm.png
+      :scale: 50 %
+      :alt: Use custom image while creating Clear Linux VM instance
+
+   .. figure:: figures/31-select-boot-disk.png
+      :scale: 50 %
+      :alt: Select Clear Linux boot disk to create a VM instance
+
+#. |CL| does not allow SSH login with a root account by default.
+   As a result, you are required to configure the VM instance with
+   your SSH public key, so that you are able to remote access to it
+   in the cloud.
+   Expand the *Management, security, disks, networking, sole tenancy* group
+   at the bottom of the VM instance creation page, copy and paste
+   your SSH key to the *Security* tab to complete the VM instance creation.
+
+   .. figure:: figures/40-ssh-key.png
```

Comment:
Suggest removing/editing out email and username as much as possible in screenshot 40. One trick could be to scroll to the top of the SSH key so it's very clear the characters in that box are an SSH key (guessing it starts with *ssh-rsa*)  and hides the rest. 

Also, of course make sure that SSH key isn't used anywhere or change up the characters :)

Reply:
Instead of re-taking the screenshot, I decided to applied blur effect to the key. :)